### PR TITLE
Added command to install php7.1-fpm for centos 7

### PR DESCRIPTION
**Pull request by asrar7787**
Added command to install php7.1-fpm for centos 7

## This PR is a:

- [x ] New topic
- [√] Content update
- [x ] Content fix or rewrite
- [x ] Bug fix or improvement

## Summary

1. Added command to install PHP7.1-FPM on Centos 7.
2. Updated `nginx` user and group.

This PR will
1. Add command to install PHP7.1-FPM on Centos 7.
2. Update `nginx` user and group.

### CODE REVIEW

**Comment by magento-engcom-team** [comment]
@asrar7787 thank you for contributing. Please accept Community Contributors team invitation [here](https://github.com/orgs/magento/invitation) to gain extended permissions for this repository.

File: guides/v2.2/install-gde/prereq/nginx.md [hunk 1]
```diff
@@ -218,6 +218,9 @@ Magento requires several [PHP extensions]({{ page.baseurl }}/install-gde/prereq/
 1. Install `php-fpm`:
 
 		yum -y install php70w-fpm
+   
+   If you are on Centos 7.1 and want to install PHP7.1-FPM, do:
```
Comment (shrielenee) [inline guides/v2.2/install-gde/prereq/nginx.md]: Remove ", do" before the colon.

Comment (shrielenee) [reply guides/v2.2/install-gde/prereq/nginx.md]: Please also make this change in the v2.3 version of this file.

**Comment by shrielenee** [comment]
Hey @asrar7787 - Did you see my requested changes? Thanks!

File: guides/v2.2/install-gde/prereq/nginx.md (hunk 1 above)
Comment (asrar7787) [reply guides/v2.2/install-gde/prereq/nginx.md]: @shrielenee 

k will update this sometimes today

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/devdocs?pullRequest=3238) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dev.magento.com/magento/devdocs?pullRequest=3238) before we can accept your contribution.<br/>

File: guides/v2.2/install-gde/prereq/nginx.md (hunk 1 above)
Comment (asrar7787) [reply guides/v2.2/install-gde/prereq/nginx.md]: @shrielenee 

Updated please accept and merge this. I will update the v2.3 later on.

Comment (shrielenee) [reply guides/v2.2/install-gde/prereq/nginx.md]: I'll wait until you add the v2.3 changes to this PR. Thanks in advance!

**Comment by magento-cicd2** [comment]
Can one of the admins verify this patch?

File: guides/v2.2/install-gde/prereq/nginx.md (hunk 1 above)
Comment (asrar7787) [reply guides/v2.2/install-gde/prereq/nginx.md]: @shrielenee 

Updated for v2.3 and also amend on nginx user and group in both PRs

**Comment by jeff-matthews** [comment]
Run tests

**Comment by jeff-matthews** [comment]
@shrielenee, I rekcommend closing this PR since it's been over almost a month since we heard from the author.

**Comment by asrar7787** [comment]
@jeff-matthews 
Sorry for late reply. Please clarify what you mean by `Run tests`

**Comment by jeff-matthews** [comment]
@asrar7787, that's a trigger phrase we use to run tests on a PR and only admins can run tests. You don't have to run tests, but please push changes per @shrielenee review so we can finish processing this PR. Thanks!

File: guides/v2.2/install-gde/prereq/nginx.md (hunk 1 above)
Comment (shrielenee) [reply guides/v2.2/install-gde/prereq/nginx.md]: Hey @asrar7787 - We still need you to update the 2.3 version. If you do that and push changes I'll merge this! Let me know.

**Comment by shrielenee** [comment]
Hello @asrar7787 - We haven't heard from you regarding this issue in two weeks. Due to inactivity, we are closing this issue. If you have further comments and feedback, please create a new issue. Thanks for taking the time to create a GitHub issue for docs improvement!

**Pull request closed without merging**